client/Elements/Components/CreateTopic/TopicAdvanceConfig.patternfly.tsx

```diff
@@ -87,59 +86,57 @@ export const TopicAdvanceConfig: React.FunctionComponent<ITopicAdvanceConfig> =
               id='advanced-create-topic'
               className='topics-wizard-content'
             >
-              <TopicContextProvider>
```

Comment:
Removing provider from `TopicAdvanceConfig` is throwing errors for CreateTopicAdvnced page, as the CoreConfiguration, MessageSEction, etc are using the topic context, we need to place the provider in a more suitable place.

Reply:
Moved the provider to bootstrap. @dlabaj just wanted to check, if there could be a better approach for handling this?